src/sql/SqlService.ts

```diff
@@ -42,6 +42,92 @@ import {HzLocalTime, HzLocalDate, HzLocalDateTime, HzOffsetDateTime} from './Dat
 
 /**
  * SQL Service of the client. You can use this service to execute SQL queries.
+ *
+ * The service is in beta state. Behavior and API might change in future releases. Binary compatibility is not
```

Comment:
Binary compatibility is a concept in Java. I don't think there is an equivalent for Node.js. I think we should delete that sentence from here.